src/__tests__/getJsonPathForPosition.spec.ts

```diff
@@ -0,0 +1,53 @@
+import * as fs from 'fs';
+import { join } from 'path';
+import { getJsonPathForPosition } from '../getJsonPathForPosition';
+import { parseWithPointers } from '../parseWithPointers';
+
+const petStore = fs.readFileSync(join(__dirname, './fixtures/petstore.oas2.yaml'), 'utf-8');
+const simple = `hello: world
+address:
+  street: 123`;
+
+describe('getJsonPathForPosition', () => {
+  describe('simple fixture', () => {
+    const result = parseWithPointers(simple);
+
+    test.each`
+      line | character | path
+      ${0} | ${0}      | ${['hello']}
+      ${0} | ${13}     | ${undefined}
+      ${0} | ${4}      | ${['hello']}
+      ${1} | ${0}      | ${['address']}
+      ${1} | ${12}     | ${['address']}
+      ${2} | ${4}      | ${['address', 'street']}
```

Comment:
sorry if this is a noob question but, why are all the numbers in `${}`?

Reply:
So that they are interpolated correctly. It's a tagged template literal, so if you don't provide template literals, the function 'won't receive them', if you know what I mean.